internal/provider/virtualmachine/resource_virtualmachine_constructor.go

```diff
@@ -173,6 +175,9 @@ func (c *Constructor) Setup() util.Processors {
 					vmBuilder.WaitForLease(interfaceName)
 				}
 				vmBuilder.NetworkInterface(interfaceName, interfaceModel, interfaceMACAddress, interfaceType, networkName)
+				if bootOrder != 0 {
+					vmBuilder.SetNetworkInterfaceBootOrder(interfaceName, uint(bootOrder)) // nolint: gosec
```

Comment:
We should check whether there is duplicated bootOrder. The KubeVirt webhook rejects request for this situation.

```
admission webhook "virtualmachine-validator.kubevirt.io" denied the request: more than one interface is connected to a pod network in spec.template.spec.interfaces, Boot order for spec.template.spec.domain.devices.interfaces[0].bootOrder already set for a different device., Boot order for spec.template.spec.domain.devices.interfaces[1].bootOrder already set for a different device.
```